code/modules/fish/fish_tank.dm

```diff
@@ -569,6 +575,11 @@
 		else
 			examine_message += "<span class = 'notice'>The lid is open.</span>"
 
+	if(automated)
+		examine_message += "<br>"
+		var/flavor_text = pick("burps and gulps", "cleans and pinks", "boops and beeps", "gloops and loops")
```

Comment:
and pinks?

Reply:
Pinks could mean decorate. So it's "cleans and decorates".